omod/src/main/resources/config.xml

```diff
@@ -84,6 +84,9 @@
         <require_module>
             org.openmrs.module.uiframework
         </require_module>
+        <require_module version="${sync2Version}">
```

Comment:
@echiteri use aware of so that when the module is stopped things do not break

Reply:
@ssmusoke noted